ifrs17/Import/ImportStorage.ipynb

```diff
@@ -251,11 +254,15 @@
                 "\n        ",
                 "\n        //SetPartition to current Period",
                 "\n        await querySource.Partition.SetAsync<PartitionByReportingNodeAndPeriod>(TargetPartition);",
-                "\n        ",
-                "\n        if(primaryScope.Select(dn => DataNodeDataBySystemName[dn]).Any(dnd => dnd.Year == args.Year) && ImportFormat != ImportFormats.Opening)",
+                "\n",
+                "\n        var primaryScopeAtInceptionYear = allImportScopesAtInceptionYear.Where(dn => primaryScope.Contains(dn));",
```

Comment:
These are two hashset. Would an Intersect perform better? @dtrzesniak

Reply:
After a brief research, it seems that Contains performs better than Intersect when working with Hashsets. @dtrzesniak @dcolleoni